base/applications/mspaint/lang/ro-RO.rc

```diff
@@ -105,18 +105,18 @@ ID_POPUPMENU MENU
 BEGIN
     POPUP "Popup #0"
     BEGIN
-        MENUITEM "C&ut\tCtrl+X", IDM_EDITCUT
-        MENUITEM "Copy\tCtrl+C", IDM_EDITCOPY
-        MENUITEM "&Paste\tCtrl+V", IDM_EDITPASTE
-        MENUITEM "De&lete selection\tDel", IDM_EDITDELETESELECTION
-        MENUITEM "Select &All\tCtrl+A", IDM_EDITSELECTALL
+        MENUITEM "&Decupează\tCtrl+X", IDM_EDITCUT
+        MENUITEM "Copiază\tCtrl+C", IDM_EDITCOPY
+        MENUITEM "&Lipește\tCtrl+V", IDM_EDITPASTE
+        MENUITEM "Ște&rge selecția\tDel", IDM_EDITDELETESELECTION
+        MENUITEM "Selectea&ză &tot\tCtrl+A", IDM_EDITSELECTALL
         MENUITEM SEPARATOR
-        MENUITEM "C&opy to...", IDM_EDITCOPYTO
-        MENUITEM "Paste &from...", IDM_EDITPASTEFROM
+        MENUITEM "C&opiază în...", IDM_EDITCOPYTO
+        MENUITEM "Lipește &din...", IDM_EDITPASTEFROM
```

Comment:
You assigned "D" action key to "Decupează". Please find a solution where options in the same menu receive different action keys. ("E" is free for the taking.)
Also, (something I haven't noticed before is that) when it's about options that does not have an immediate actions (and are marked by the trailing ellipsis in their names), please use long infinitive. See [Romanian translation notes pt. №4](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt#L16). So, in this case, instead of "Copiază în..." and "Lipește din..." it should be "Copiere în..." and "Lipire din..." respectively.